sktime/transformations/panel/summarize/_extract.py

```diff
@@ -17,18 +22,53 @@
 class PlateauFinder(BaseTransformer):
     """Plateau finder transformer.
 
-    Transformer that finds segments of the same given value, plateau in
-    the time series, and
-    returns the starting indices and lengths.
+    Transformer that finds segments of the same given value, aka plateau(s) in
+    the time series, and returns the starting indices and lengths. It expects a
+    dataframe with one column of nested Series and it extracts the plateau for
```

Comment:
the format assumption is not correct - you can pass non-nested data frames to the transformers, too.
I assume the features are computed by series. Can you simply detail what happens for a single series, e.g., a single `pd.Series` object (not a nested data frame)? That would be easier to understand.

It should work out of the box.

Reply:
mmm not sure what you mean... If the you pass a dataframe like this (where the timeseries is spread into multiple columns ) is not going to work:
|    |   0 |   1 |   2 |   3 |   4 |   5 |   6 |   7 |
|---:|----:|----:|----:|----:|----:|----:|----:|----:|
|  0 |  -1 |  -1 |   3 |   3 |  -1 |   2 |   2 |   3 |
|  1 |   0 |  -1 |  -1 |  -1 |  -1 |  -1 |   2 |  -1 |
|  2 |   2 |  -1 |  -1 |  -1 |   2 |  -1 |   3 |   1 |
|  3 |   1 |  -1 |  -1 |   3 |  -1 |  -1 |   2 |   0 |
Output:
```bash
X must be in an sktime compatible format, of scitype Series, Panel or Hierarchical, for instance
a pandas.DataFrame with sktime compatible time indices, or with MultiIndex and lowest level
a sktime compatible time index. allowed compatible mtype format specifications
are: ['pd.Series', 'pd.DataFrame', 'np.ndarray', 'nested_univ', 'numpy3D', 'pd-multiindex',
'df-list', 'pd_multiindex_hier'] See the data format tutorial examples/AA_datatypes_and_datasets.ipynb
```

Reply:
I mean:

* I think the plateaufinder transform is applied per series, in a panel
* can you please write in the docstring what precisely happens, for a series `y(t_0), ..., y(t_k)`?

Reply:
I'm not sure what type the plateau finder will accept but I will check now. I suppose you are talking about the "columns" of this dataframe (the one you call "panel"):
![image](https://user-images.githubusercontent.com/34258464/164119605-b5878715-0c60-43d4-bc69-3acbb9904f4a.png)
While the plateau example that is already in the code base [here](https://github.com/alan-turing-institute/sktime/blob/main/examples/plateau_finder.ipynb), I think, use a type of `nested_univ`. And that is the one I based the docstring on.

I will write in the docstring of what happens to a series of timepoints for one "instance", then will test a couple of more input data the `PlateauFinder` can work with. How does that sound?

Reply:
I can confirm that the same example data, reshaped as panel with multindex like this below are not working in the when calling `fit_transform` over it.
![image](https://user-images.githubusercontent.com/34258464/164123215-b3e670f3-f01f-4172-8585-456844d1faa2.png)
Output error:
```bash
X must be in an sktime compatible format, of scitype Series, Panel or Hierarchical, for instance
a pandas.DataFrame with sktime compatible time indices, or with MultiIndex and lowest level
a sktime compatible time index. allowed compatible mtype format specifications
are: ['pd.Series', 'pd.DataFrame', 'np.ndarray', 'nested_univ', 'numpy3D', 'pd-multiindex',
'df-list', 'pd_multiindex_hier'] See the data format tutorial
examples/AA_datatypes_and_datasets.ipynb
```
Neither this works:
![image](https://user-images.githubusercontent.com/34258464/164123668-4aa77459-9fa7-411e-98f4-1687f701cddd.png)
Error:
```bash
X must be univariate but is not
```
So I assume the `PlateauFinder` wants the data shapes as `nested_univ` with a single column....?

Reply:
can you paste some code please?